pennylane/transforms/qcut.py

```diff
@@ -941,6 +940,33 @@ def circuit(x):
         >>> qml.grad(cut_circuit)(x)
         -0.506395895364911
     """
+    if len(tape.measurements) != 1:
+        raise ValueError(
+            "The circuit cutting workflow only supports circuits with a single output "
+            "measurement"
+        )
+
+    if not all(m.return_type is Expectation for m in tape.measurements):
+        raise ValueError(
+            "The circuit cutting workflow only supports circuits with expectation "
+            "value measurements"
+        )
+
+    if use_opt_einsum:
+        try:
+            import opt_einsum  # pylint: disable=import-outside-toplevel,unused-import
+        except ImportError as e:
+            raise ImportError(
+                "The opt_einsum package is required when use_opt_einsum is set to "
+                "True in the cut_circuit function. This package can be "
+                "installed using:\npip install opt_einsum"
+            ) from e
+
+    num_cut = len([op for op in tape.operations if isinstance(op, WireCut)])
+    if num_cut == 0:
+        raise ValueError(
+            "Attempting to apply the circuit cutting workflow to a circuit without any cuts"
```

Comment:
Is the workflow actually attempted? Maybe `"Cannot apply the circuit cutting workflow to a circuit without any cuts"`?

Reply:
Nice! Fixed here: 2fb04e25b630c4da5214a446aedccccdb5bbaef9